code-challenges.js

```diff
@@ -28,3 +69,21 @@ var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
 var hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
+
+//create a function that takes in array of numbers
+
+const fullHouse = (array) => {
+if(array[0] === array[1] && array[1] != array[2] && array.length >= 3){
+            return false
+        }
+        else if(array[0] != array[1] && array[1] === array[2] && array.length >= 3){
+            return false
+        }
+        else if(array[0] != array[1] && array[1] != array[2] && array[2] === array[3] && array.length >= 3){
+            return true
+        } else {
+          return true
+        }
+}
```

Comment:
cool! so this solution works with the test hands we gave you. but doesnt work with all poker hands. this one will need to be refactored. but you're on a good track! let me know if you want to do this one together!